primitives/fee-handler/src/fee_handler.rs

```diff
@@ -0,0 +1,370 @@
+use crate::withdraw_credit::WithdrawCredit;
+use core::marker::PhantomData;
+use frame_support::{
+	pallet_prelude::DispatchError,
+	traits::{fungible, fungibles, ConstU32, Defensive, Imbalance},
+	BoundedVec,
+	__private::log,
```

Comment:
This import should be from the dependency directly no from `frame_support`

Reply:
Ah, I was lazy here because I don't depend on it yet. XD. Will fix it.